src/hdusd/bl_nodes/nodes/shader.py

```diff
@@ -291,11 +291,11 @@ def enabled(val):
         # Emission -> Emission
         if enabled(emission):
             # more related formula for emission weight:
-            emission_weight = emission.average_xyz().min(1.0) * 0.5 + 0.5
+            emission_weight = min(self.node.inputs["Emission Strength"].default_value, 1.0) * 0.5
 
             result.set_inputs({
                 'uber_emission_weight': emission_weight,
-                'uber_emission_color': emission,
+                'uber_emission_color': emission.data[:3],
```

Comment:
this is incorrect, because emission NodeItem object could be a node or float. 
Same for emission_weight, we expect not only float, but node also.